README.md

```diff
@@ -1,5 +1,44 @@
 # vue-router-next [![CircleCI](https://circleci.com/gh/vuejs/vue-router-next.svg?style=svg)](https://circleci.com/gh/vuejs/vue-router-next)
 
+## Install as a package for your vue3 project
+
+```bash
+npm i @posva/vue-router-next --save
```

Comment:
Where did you find these? 😄 it's just `vue-router@next`

Reply:
Ahhh gotcha. I found it here on npmjs. 😆 
https://www.npmjs.com/package/@posva/vue-router-next